jmslib/src/main/java/com/redhat/mqe/lib/SenderClient.java

```diff
@@ -20,6 +20,7 @@
 package com.redhat.mqe.lib;
 
 import com.redhat.mqe.lib.message.MessageProvider;
+import sun.reflect.Reflection;
```

Comment:
Do you need this? You do not use this class anywhere. Did you add it just because of the compilation fail on java 11?

Reply:
Nope, I forgot to remove it. Updating.